src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -135,6 +135,7 @@ function ReportPreview({
     const isMoneyRequestReport = ReportUtils.isMoneyRequestReport(iouReport);
     const transactionsWithReceipts = ReportUtils.getTransactionsWithReceipts(iouReportID);
     const numberOfScanningReceipts = transactionsWithReceipts.filter((transaction) => TransactionUtils.isReceiptBeingScanned(transaction)).length;
+    const numberOfPendingReceipts = transactionsWithReceipts.filter((transaction) => TransactionUtils.isPending(transaction) && TransactionUtils.isCardTransaction(transaction)).length;
```

Comment:
Is it considered as pending receipt when `cardID` exists in `transaction`?
Also, should then name be `numberOfPendingRequests`? or this filters receipts only?

Reply:
good catch, the variable name should be `numberOfPendingRequests`